src/Csharp/packages/base/src/NativeUtils.cs

```diff
@@ -180,7 +181,25 @@ public static float[] FlattenVectorArray(IEnumerable<Vector3D> vectors)
             The number of elements in result_array is not equal to the number of
             elements in float_array / 3
        */
-		public static Vector3D[] FloatArrayToVectorArray(float[] float_array, bool[] result_array)
+
+        public static float[] FlattenVectorArrayUnsafe(Vector3D[] vectors)
```

Comment:
Could you confirm that the unittests are passing after you made this change? I haven't looked at this much yet, but it seems to be changing the API?

Reply:
Ah, my bad. I wanted to paste this under this method:
https://github.com/cadop/dhart/blob/44d2d9a52a18c4355c226116522bd7be99150f1a/src/Csharp/packages/base/src/NativeUtils.cs#L137-L160

I'm keeping it untouched and just introducing another overload working with Vector3D[]
![image](https://user-images.githubusercontent.com/49192999/205091771-43c33db2-beb3-4daa-9257-cf6468e1c172.png)

I'll put it in the right place